src/style/index.scss

```
@@ -277,7 +297,7 @@ hr.spacer {
 }
 
 .fees-abbrev-underline {
-    border-bottom: 1px dashed currentColor;
+    border-bottom: 1px dotted currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)